.npmignore

```diff
@@ -1,14 +1,16 @@
+# IDE
 .idea/
 .vscode/
 
 # development
 src/
-test/
+tests/
 
-# Ignore map sources
-*.js.map
```

Comment:
Would you like to upload the .js.map files to npm?

Reply:
there some benefits to include map files for debugging, but wee decided to not generate the  map files in publish transpile so there isn`t files to ignore

Reply:
i revert this if you want